hack/make.sh

```diff
@@ -174,6 +174,16 @@ if [ "$(uname -s)" = 'FreeBSD' ]; then
 	LDFLAGS="$LDFLAGS -extld clang"
 fi
 
+if [ "$(uname -s)" = 'darwin' ]; then
+	# Tell cgo the compiler is Clang, not GCC
+	# https://code.google.com/p/go/source/browse/src/cmd/cgo/gcc.go?spec=svne77e74371f2340ee08622ce602e9f7b15f29d8d3&r=e6794866ebeba2bf8818b9261b54e2eef1c9e588#752
```

Comment:
if this isn't real issue (coz on macOS gcc and clang are same), let's try to revert.